Exiled.Events/Patches/Events/Map/SpawningItem.cs

```diff
@@ -0,0 +1,46 @@
+// -----------------------------------------------------------------------
+// <copyright file="SpawningItem.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Patches.Events.Map
+{
+    using System.Collections.Generic;
+#pragma warning disable SA1313
+
+    using Exiled.API.Features;
+    using Exiled.Events.EventArgs;
+
+    using HarmonyLib;
+
+    using MEC;
+
+    using UnityEngine;
+
+    /// <summary>
+    /// Patches <see cref="RandomItemSpawner.SpawnerItemToSpawn.DoorTrigger"/>.
+    /// Adds the <see cref="Handlers.Map.SpawningItem"/> event.
+    /// </summary>
+    [HarmonyPatch(typeof(RandomItemSpawner.SpawnerItemToSpawn), nameof(RandomItemSpawner.SpawnerItemToSpawn.Spawn))]
+    internal static class SpawningItem
+    {
+        private static IEnumerator<float> Prefix(RandomItemSpawner.SpawnerItemToSpawn __instance)
```

Comment:
This won't work, it you want to implement that kind of patch then it must be a Postfix and the first parameter must be `IEnumerator<float> values` (keep in mind, harmony is name sensitive)

Reply:
Anyway, the requirements were in 2.0.1, correct me if I'm wrong and it works